firstvoices/backend/resources/base.py

```diff
@@ -38,6 +44,9 @@ def before_import(self, dataset, **kwargs):
         dataset.append_col(lambda x: str(self.site.id), header="site")
         dataset.append_col(lambda x: str(self.run_as_user), header="created_by")
         dataset.append_col(lambda x: str(self.run_as_user), header="last_modified_by")
+        dataset.append_col(
+            lambda x: str(self.run_as_user), header="system_last_modified_by"
```

Comment:
For the system modified user, if possible we should use the person actually submitting the request instead of the value of the run_as_user